modules/dop/component-protocol/components/auto-test-space-list/components/spaceList/render.go

```diff
@@ -26,10 +26,10 @@ import (
 	"github.com/erda-project/erda/bundle"
 	"github.com/erda-project/erda/modules/dop/component-protocol/components/auto-test-space-list/common"
 	"github.com/erda-project/erda/modules/dop/component-protocol/components/auto-test-space-list/i18n"
+	text "github.com/erda-project/erda/modules/dop/component-protocol/components/common"
```

Comment:
why text

Reply:
alias of text content, another common package has already been used